README.md

```diff
@@ -1,273 +1,88 @@
 ```
-
-And then call the prediction endpoint 
-
-```bash
-curl http://127.0.0.1:8080/predictions/densenet161 -T kitten_small.jpg
+docker pull pytorch/torchserve
 ```
 
-Which will return the following JSON object
-
-```json
-[
-  {
-    "tiger_cat": 0.46933549642562866
-  },
-  {
-    "tabby": 0.4633878469467163
-  },
-  {
-    "Egyptian_cat": 0.06456148624420166
-  },
-  {
-    "lynx": 0.0012828214094042778
-  },
-  {
-    "plastic_bag": 0.00023323034110944718
-  }
-]
-```
-
-All interactions with the endpoint will be logged in the `logs/` directory, so make sure to check it out!
-
-Now you've seen how easy it can be to serve a deep learning model with TorchServe! [Would you like to know more?](docs/server.md)
-
-### Stop TorchServe
-
-To stop the currently running TorchServe instance, run:
-
-```bash
-torchserve --stop
-```
-
-### Inspect the logs
-All the logs you've seen as output to stdout related to model registration, management, inference are recorded in the `/logs` folder.
-
-High level performance data like Throughput or Percentile Precision can be generated with [Benchmark](benchmark/README.md) and visualized in a report.
-
-### Concurrency And Number of Workers
-TorchServe exposes configurations that allow the user to configure the number of worker threads on CPU and GPUs. There is an important config property that can speed up the server depending on the workload.
-*Note: the following property has bigger impact under heavy workloads.*
-
-**CPU**: there is a config property called `workers` which sets the number of worker threads for a model. The best value to set `workers` to is to start with `num physical cores / 2` and increase it as much possible after setting `torch.set_num_threads(1)` in your handler.
-
-**GPU**: there is a config property called `number_of_gpu` that tells the server to use a specific number of GPUs per model. In cases where we register multiple models with the server, this will apply to all the models registered. If this is set to a low value (ex: 0 or 1), it will result in under-utilization of GPUs. On the contrary, setting to a high value (>= max GPUs available on the system) results in as many workers getting spawned per model. Clearly, this will result in unnecessary contention for GPUs and can result in sub-optimal scheduling of threads to GPU.
-```
-ValueToSet = (Number of Hardware GPUs) / (Number of Unique Models)
-```
-
-## Quick Start with Docker
 Refer to [torchserve docker](docker/README.md) for details.
 
-## Highlighted Examples
-* [HuggingFace Transformers](examples/Huggingface_Transformers)
+## ⚡ Why TorchServe
+* [Model Management API](docs/management_api.md): multi model management with optimized worker to model allocation
+* [Inference API](docs/inference_api.md): REST and gRPC support for batched inference
+* [TorchServe Workflows](examples/Workflows/README.md): deploy complex DAGs with multiple interdependent models
```

Comment:
@msaroufim how about adding the ensemble(term) somewhere in the doc to make sure it relates to user terminology.

Reply:
I'm not sure how I feel about this because ensemble to me means aggregating the predictions of `n` models into a single one whereas a workflow is about setting up a DAG which is more general and what we support

Reply:
@msaroufim right, as it supports parallel workflow so it may fit into that defintion as well?